packages/plugin-typescript/sources/index.ts

```diff
@@ -1,11 +1,12 @@
-import {Descriptor, Plugin, Workspace, ResolveOptions, Manifest, AllDependencies, DescriptorHash, Package} from '@yarnpkg/core';
-import {structUtils, ThrowReport, miscUtils, semverUtils}                                                  from '@yarnpkg/core';
-import {Hooks as EssentialsHooks}                                                                          from '@yarnpkg/plugin-essentials';
-import {suggestUtils}                                                                                      from '@yarnpkg/plugin-essentials';
-import {Hooks as PackHooks}                                                                                from '@yarnpkg/plugin-pack';
-import semver                                                                                              from 'semver';
+import {Descriptor, Plugin, Workspace, ResolveOptions, Manifest, AllDependencies, DescriptorHash, Package, SettingsType} from '@yarnpkg/core';
+import {structUtils, ThrowReport, miscUtils, semverUtils}                                                                from '@yarnpkg/core';
```

Comment:
NIT: Would be nice if our linting setup saw that it could place `SettingsType` in the second import to avoid all these whitespace changes